package.json

```diff
@@ -47,5 +49,14 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Snyk/resolve-deps.git"
+  },
+  "nyc": {
+    "exclude": [
+      "dist",
```

Comment:
the new dist folder was added, as it should be ignored from coverage.
All the rest are the default values, which now need to be stated.